app/service/update-appeal-service.ts

```diff
@@ -79,6 +79,7 @@ export default class UpdateAppealService {
       appealLastModified: ccdCase.last_modified,
       application: {
         homeOfficeRefNumber: caseData.homeOfficeReferenceNumber,
+        isPartiallySaved: !!caseData.homeOfficeReferenceNumber,
```

Comment:
Is !! something special? or have you negated this twice, is that so you get a boolean?